src/logic/tokens/utils/tokensStorage.js

```diff
@@ -24,6 +25,20 @@ export const getActiveTokens = async (): Promise<Object<string, TokenProps>> =>
   return data || {}
 }
 
+export const saveBlacklistedTokens = async (tokens: Map<string, Token>) => {
```

Comment:
why do we store them in a separate key? Aren't they stored individually for each safe in safe's record?

Reply:
So we store active tokens for all safes because their data should always be there for better UX. But I think for blacklisted ones it doesn't make much sense

Reply:
@fernandomg what about this?

Reply:
@mikheevm, sorry for the late response. You can see the changes in https://github.com/gnosis/safe-react/pull/300/commits/b0791301e464ad302707282342f0f59f1ef636cd

By deleting the action to persist _blakclisted tokens_ into localStorage it finished working. It took me a while to get around it and understand what I was doing wrong.

Let me know if there's anything else to fix or change.

Thank you!

Reply:
Thanks :)